docs/source/chainermn/installation/guide.rst

```diff
@@ -110,16 +110,17 @@ We tested ChainerMN on all the following environments.
 * OS
   
   * Ubuntu 14.04 LTS 64bit
+  * Ubuntu 16.04 LTS 64bit
 
 * Python 2.7.13 3.5.1 3.6.1
-* Chainer 3.5.0 4.0.0 4.1.0
-* CuPy 2.5.0 4.0.0 4.1.0
+* Chainer 3.5.0 4.4.0
```

Comment:
I think we should remove chainer versions, because ChainerMN is now bundled to Chainer and "supported chainer versions" does not make much sense.
(That's my mistake.)

Reply:
That's true. I did that in #5302.

Reply:
thanks!